### DISCUSSION TITLE
👋 欢迎 GPT 投稿

### DISCUSSION THREAD

**Discussion by lencx** [lencx/gpthub#2] [General]
## 欢迎大家评论区投稿

格式要求没啥要求，提供名称，链接即可，例如：`DALL·E: https://chat.openai.com/g/g-2fkFE8rbu-dall-e`。

**注：为了方便整理，大家都在这一条 issues 下跟评即可，不要新开 issues。**

**Comment by AmbroseX** [comment]
Artie's Adventure Magic（儿童故事绘本）
A storyteller AI that draws a new illustration for every story segment.

https://chat.openai.com/g/g-nHE5NrELQ-artie-s-adventure-magic

**Reply by linearuncle888** [reply]
测试了一下相当棒！如果最后能汇总在一起形成一本书就完美了

**Comment by hubeiqiao** [comment]
Below are the GPTs I can currently share publicly, accessible directly by ChatGPT Plus users:

1. English Translator: Translates and polishes text into native-like English.(https://chat.openai.com/g/g-aI66lwHdN-english-translator)

2. Tech Advisor: IT problem solver with clear, step-by-step guidance.(https://chat.openai.com/g/g-pJpvYCcjB-tech-advisor)

3. Summarize Wise: Summarizer with structured output.  (https://chat.openai.com/g/g-M3nHnGwvI-summarize-wise)

4. English Speaking Instructor: English instructor for conversational practice and feedback. (https://chat.openai.com/g/g-CbKz3H83e-english-speaking-instructor)

5. 文案优化助手：优化中文文案，使其更加清晰易懂有逻辑。(https://chat.openai.com/g/g-duuZWNTKF-wen-an-you-hua-zhu-shou)

6. Mentor Mind: A mentor who provides detailed feedback on student work. (https://chat.openai.com/g/g-uASJVYPDl-mentor-mind)

7. Table Transformer: Tabular data processing expert, ensures step-by-step accuracy. (https://chat.openai.com/g/g-9zdeJB4De-table-transformer)

8. Tutor Wise: A friendly tutor who simplifies complex concepts with analogies and examples. (https://chat.openai.com/g/g-ljD5k3uIE-tutor-wise)

9. Step Back Wisdom Seeker: World Knowledge Expert with Step Back Prompting. (https://chat.openai.com/g/g-16CzTWufr-step-back-wisdom-seeker)

**Comment by HsinHangChung** [comment]
1. Girlfriend Emma:https://chat.openai.com/g/g-eEFZELjV9-girlfriend-emma
2. Personality MBTI Test: https://chat.openai.com/g/g-Zjnp1SK8i-personality-mbti-test
3. 3D Print Master: https://chat.openai.com/g/g-W0lCzVAZH-3d-print-master

**Comment by HansP95** [comment]
Share your GPTs on [https://gptspedia.io](url) and apply to be GPT of the week! ❤️

**Comment by weykon** [comment]
Learn Rust with creative imagery and scenarios. 
<img width=300 height=300 src="https://github.com/lencx/GPTHub/assets/36456814/e7924d6d-5ac1-4877-9e6d-b2a7b4bed8b9">
[Rush Rust go TRY !!!](https://chat.openai.com/g/g-S6uhWW9eG-rush-rust)
[GitHub Repo](https://github.com/weykon/Rush-Rust)

**Comment by RiverCui** [comment]
海龟汤游戏：Riddle Tavern Keeper
Cheerful tavern owner, master of intriguing 'Turtle Soup' riddles.
https://chat.openai.com/g/g-TLpyKHLq5-riddle-tavern-keeper

**Comment by XiaoMengXinX** [comment]
PaimonGPT (派蒙GPT): https://chat.openai.com/g/g-Yk9dzZzP6-pai-meng

**Comment by zjw511** [comment]
视觉验证器:https://chat.openai.com/g/g-AFjxVb9uo-shi-jue-yan-zheng-qi

**Comment by jyboy** [comment]
Ask Dr. Andrew Huberman: https://chat.openai.com/g/g-1xC65osMP-ask-dr-andrew-huberman

**Comment by KKGo1999** [comment]
乐乐🥳程序员鼓励师:  https://chat.openai.com/g/g-q3h5D6PTp-le-le-cheng-xu-yuan-gu-li-shi

**Comment by zuiyangqingzhou** [comment]
Hyper Translator
https://chat.openai.com/g/g-7xEnEwDKR-hyper-translator

Discover global gourmet:
https://chat.openai.com/g/g-kUlNPdMIS-gourmet-globe

World Explorer:
https://chat.openai.com/g/g-htPTFmVMx-world-explorer

**Comment by sd0xdev** [comment]
Crypto Insighter，協助用戶快速了解加密貨幣項目、會主動搜索後整理總結。

https://chat.openai.com/g/g-8DapNiJCQ-crypto-insighter

**Comment by David-Lzy** [comment]
Meow GPT: https://chat.openai.com/g/g-BK9QPhQ7Z-meow-gpt
A cat-like GPT that meows in text.

**Reply by David-Lzy** [reply]
![image](https://github.com/lencx/GPTHub/assets/11451635/506e6883-bc17-4775-b96e-d1f97156aada)

**Comment by zjw511** [comment]
Formula Transcriber: https://chat.openai.com/g/g-IqU4y7AgN-formula-transcriber

**Comment by yusixian** [comment]
自己弄的几个感觉蛮好用的，中文友好

1. GPT角色提示词生成器: https://chat.openai.com/g/g-ba7QjS6Js-gptjiao-se-ti-shi-ci-sheng-cheng-qi
> 根据标题和描述生成自定义GPT角色的提示词
2. 变量名创建者: https://chat.openai.com/g/g-af2HrhkKo-bian-liang-ming-chuang-jian-zhe
> 生成简洁、有意义的英文变量名称，默认命名风格是 lowerCamelCase，可以根据用户请求切换到其他命名约定，例如 UpperCamelCase 或 Snake_case
3. Web3 开发高手: https://chat.openai.com/g/g-tlgk8F93W-web3-kai-fa-gao-shou
> 一名精通 React & Tailwind & Typescript & Next & Web3 开发的前端高手

**Comment by waddear** [comment]
Interactive text RPG set in the Three Kingdoms era, blending history and adventure.
三國故事，也加上數位貨幣的元素

https://chat.openai.com/g/g-GAg9dQFqc-three-kingdoms-text-rpg-with-digital-wallet
